apps/src/applab/applab.js

```diff
@@ -392,6 +386,7 @@ Applab.init = function(config) {
   Applab.generatedProperties = {
     ...config.initialGeneratedProperties
   };
+  //remove
```

Comment:
Is this comment intended to leave? If we want to leave it here, we might want to add additional context about what is being removed, why, and when.

Reply:
Oh, found it here! NVM! https://github.com/code-dot-org/code-dot-org/pull/48224/files#diff-0dc785e6d0108a24cc4e8ca5d7e0359dd8387cf9c620812d438f4ca5f0c656eeL389

Reply:
Yeah didn't mean to leave this, but I do remove it in the next PR.